### PR TITLE
Feat/landing faq section

### PR DESCRIPTION
**Pull request by meczajkowski**
As there is no landing page yet, I put the FAQ-related stuff into `@/components/LandingPage/FAQ` .
I think it would be a good idea to put every section into  `@/components/LandingPage/(section)` as it would be very clear where everything related to the Landing Page lives.

I am not sure about `@/components/LandingPage/FAQ/Accordion/icons` and `@/components/LandingPage/FAQ/enums.ts` if this is a good approach. I did not find a better place for them to live.

Also I was thinking about whether the questions and answers, which are hardcoded for now, could be an array of objects passed via props to `<FAQ data={faqData}/>` for example.

Currently the FAQ is not displayed in any page. To use it just put the `<FAQ />` component into the landing page.

Any suggestions are welcome.
### Desktop View

<image width="500px" src="https://github.com/nerdbord/good-dev-hunting-app/assets/110562040/367ffa9a-e90c-4ea5-b9eb-1b2d8024b7d3"/>

<image width="500px" src="https://github.com/nerdbord/good-dev-hunting-app/assets/110562040/239eb4e1-a41e-4439-a9e6-d6a3441cbb55"/>

### Mobile View

<image width="250px" src="https://github.com/nerdbord/good-dev-hunting-app/assets/110562040/b78e8b18-1c9a-4981-84b0-506cf94e61fc"/>
<image width="250px" src="https://github.com/nerdbord/good-dev-hunting-app/assets/110562040/c867370c-e1db-4911-970f-e84ed6371923"/>

### CODE REVIEW

File: src/components/LandingPage/FAQ/Accordion/Accordion.module.scss
Comment (Sumick) [inline src/components/LandingPage/FAQ/Accordion/Accordion.module.scss:1]: **nit:** Accordion is pretty generic component that can be used anywhere, not just in FAQ. We can put it straight into `/components/Accordion`

File: src/components/LandingPage/FAQ/FAQ.tsx
Comment (Sumick) [inline src/components/LandingPage/FAQ/FAQ.tsx:1]: **nit:** As we should only name components with PascalCase, let's change LandingPage directory into `/src/components/landing-page/FAQSection.tsx`, and `/src/components/landing-page/FAQSection.module.scss`

File: src/components/LandingPage/FAQ/enums.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export enum AccordionIconType {
```
Comment (Sumick) [inline src/components/LandingPage/FAQ/enums.ts:1]: **nit:** Let's keep it close to implementation, so inside `Accordion.tsx` file

**Review by Sumick — CHANGES_REQUESTED** [review]
Mostly look good, let's just change files placement, and dir naming a little bit

**Review by Sumick — APPROVED** [review]
🙌

**Review by wawrzonnn — APPROVED** [review]

**Review by mmcode9407 — APPROVED** [review]

**Pull request merged**